### DISCUSSION TITLE
Refine DevOps Example

### DISCUSSION THREAD

**Discussion by cf-sewe** [refinedev/refine#5371] [Q&A] [answered]
Is the Refine DevOps example code available somewhere? I would like to see a real world example, in especial with API integration for such use case. I would like to build a solution for managing cloud instances, so the DevOps example would be kind of easy to start with.

Cheers,
Sebastian

![image](https://github.com/refinedev/refine/assets/52649758/8ad67117-5e74-4198-8bb8-45f32d6665cc)

**Comment by BatuhanW** [comment, marked as answer]
Hello @cf-sewe we don't have this example available yet.

However, there is a community data provider for k8s, which simplifies building this example.

https://github.com/webzard-io/k8s-api-provider
